.gitignore

```diff
@@ -0,0 +1,4 @@
+phpunit.xml
+vendor
+build
+composer.lock
```

Comment:
@lepture  I used editorcofig and added it project

```
# editorconfig.org
root = true

# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
insert_final_newline = true
charset = utf-8

[*.php]
indent_style = space
indent_size = 4
trim_trailing_whitespace = true

[*.{js,twig,css,scss,html}]
indent_style = space
indent_size = 2

[{Gruntfile.js,bower.json,composer.json,package.json,*.yml}]
indent_style = space
indent_size = 2

[*.md]
trim_trailing_whitespace = false

```

But online is not display...